docs/contributing/index.md

```diff
@@ -29,8 +29,13 @@ committing
 
 Wagtail has internationalisation support so if you are fluent in a non-English language you can contribute by localising the interface.
 
-Translation work should be submitted through [Transifex](https://explore.transifex.com/torchbox/wagtail/).
+```{toctree}
+:maxdepth: 2
 
+translations
+```
+
+Translation work should be submitted through [Transifex](https://explore.transifex.com/torchbox/wagtail/).
 ## Other contributions
 
 We welcome contributions to all aspects of Wagtail. If you would like to improve the design of the user interface, or extend the documentation, please submit a pull request as above. If you're not familiar with Github or pull requests, [contact us directly](mailto:hello@wagtail.org) and we'll work something out.
```

Comment:
However, further down on this file, we do need to include the sub-page in the main ```{toctree}```